packages/components/Button/src/Button.types.ts

```diff
@@ -154,6 +155,7 @@ export interface ButtonInfo {
 
 export interface ButtonSlotProps {
   root: React.PropsWithRef<PressablePropsExtended>;
+  rippleContainer?: IViewProps;
```

Comment:
nit but not necessary, I'd leave a simple ` // Android only` comment around rippleContainer.